recipes/libsvm/meta.yaml

```diff
@@ -20,9 +20,13 @@ build:
     - vc10  # [win and py34]
     - vc14  # [win and py35]
 
+requirements:
+  build:
+    - python  # [win]
+
 test:
   requires:
-    - python {{ environ['PY_VER'] + '*' }}                    # [win]
+    - python  # [win]
```

Comment:
If we can switch back to using `environ['PY_VER']` here, but leave everything else alone (including build requirements) and CI passes. I would happily merge this. Thanks for trying this out here so that we can see this doesn't work.